client/App/pages/LandingPage/components/Header/VerticalCarousel/VerticalCarouselItem/VerticalCarouselItem.tsx

```diff
@@ -0,0 +1,60 @@
+import { MouseEventHandler } from 'react';
+import styles from './VerticalCarouselItem.module.scss';
+
+interface Props {
+  slide: {
+    clientName: string;
+    clientPicture: string | undefined;
```

Comment:
Same here, delete undefined.

Reply:
I removed it in the other branch, it's my fault that I created new branches from previous, not from master :(